static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -158,4 +164,74 @@
             bottom: 8px;
         }
     }
+
+    .immersive-main-media__media & {
+        // & = .youtube-media-atom
+        padding-bottom: 0;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        height: 100%;
+        z-index: 0;
+        &__immersive-interface {
+            position: absolute;
+            top: 50vh;
+            transform: translateY(-100%);
+            padding-bottom: $gs-baseline * 2;
+        }
+        &__play-button,
+        &__bottom-bar__duration {
+            display: inline-block;
+            vertical-align: middle;
+            float: none;
+            top: auto;
+            left: auto;
+            right: auto;
+            bottom: auto;
+        }
+        &__play-button {
+            position: relative;
+            width: 92px;
+            height: 92px;
+            transform: none;
+            &:hover {
+                transform: none;
+            }
+            &:before {
+                width: 100%;
+                height: 100%;
+                border-radius: 100%;
+            }
+            &:after {
+                display: none;
+            }
+            svg {
+                width: 46px;
+                height: 39px;
+                fill: $multimedia-main-1;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                transform: translateX(-42%) translateY(-50%);
+            }
+        }
+        &__bottom-bar__duration {
```

Comment:
I removed a duplicate rule but I think the block overall is necessary here as the above block affects both the play button and the duration block, for the immersive interface